pyop2/local_kernel.py

```diff
@@ -152,28 +152,9 @@ def arguments(self):
                      for acc, dtype in zip(self.accesses, self.dtypes))
 
     @cached_property
+    @abc.abstractmethod
     def num_flops(self):
```

Comment:
Can the docstring explanation for `flop_count` be improved to say something like "...if this is not provided then PyOP2 will try to estimate a FLOP count.".

Reply:
Does PyOP2 do that?

Reply:
For string kernels it returns 0, for coffee kernels it returns an estimation and for loopy kernels it actually measures the FLOPS. Is there another type of local kernel?

Reply:
No you're right. What I'm saying is that the docstring should describe what happens if `flop_count` is `None` as its behaviour is not obvious.

Reply:
Okay I added what I said here to the method description.